src/__tests__/getByApi.test.js

```diff
@@ -33,6 +28,6 @@ test('byTestId returns only native elements', () => {
   expect(getAllByTestId('view')).toHaveLength(1);
   expect(getAllByTestId('button')).toHaveLength(1);
 
-  expect(queryByTestId('myComponent')).toBeFalsy();
-  expect(queryAllByTestId('myComponent')).toHaveLength(0);
+  expect(() => getByTestId('myComponent')).toThrowError();
+  expect(() => getAllByTestId('myComponent')).toThrowError();
```

Comment:
Let's add what text we expect to see there like `toThrowError('no instances found')`. The `toThrowError()` matcher without an argument may hide unexpected errors and should generally be avoided

Reply:
Good idea, fixed! :-)